pytest/tests/sanity/state_sync_epoch_boundary.py

```diff
@@ -22,63 +22,50 @@
 
 from configured_logger import logger
 
-EPOCH_LENGTH = 50
+EPOCH_LENGTH = 30
 
 state_parts_dir = str(pathlib.Path(tempfile.gettempdir()) / 'state_parts')
 
 config0 = {
-    'gc_num_epochs_to_keep': 100,
-    'log_summary_period': {
-        'secs': 0,
-        'nanos': 500000000
-    },
-    'log_summary_style': 'plain',
-    'state_sync': {
-        'dump': {
-            'location': {
-                'Filesystem': {
-                    'root_dir': state_parts_dir
+    "state_sync": {
+        "dump": {
+            "location": {
+                "Filesystem": {
+                    "root_dir": state_parts_dir
                 }
             },
-            'iteration_delay': {
-                'secs': 0,
-                'nanos': 100000000
+            "iteration_delay": {
+                "secs": 0,
+                "nanos": 100000000
             },
         }
     },
-    'store.state_snapshot_enabled': True,
-    'tracked_shards': [0],
+    "store.state_snapshot_enabled": True,
+    "tracked_shards": [0],
 }
 config1 = {
-    'gc_num_epochs_to_keep': 100,
-    'log_summary_period': {
-        'secs': 0,
-        'nanos': 500000000
-    },
-    'log_summary_style': 'plain',
-    'state_sync': {
-        'sync': {
-            'ExternalStorage': {
-                'location': {
-                    'Filesystem': {
-                        'root_dir': state_parts_dir
+    "state_sync": {
+        "sync": {
+            "ExternalStorage": {
+                "location": {
+                    "Filesystem": {
+                        "root_dir": state_parts_dir
                     }
                 }
             }
         }
     },
-    'state_sync_enabled': True,
-    'consensus.state_sync_timeout': {
-        'secs': 0,
-        'nanos': 500000000
+    "state_sync_enabled": True,
+    "consensus.state_sync_timeout": {
+        "secs": 0,
+        "nanos": 500000000
     },
-    'tracked_shard_schedule': [[0, 2, 3], [0, 2, 3], [0, 1], [0, 1], [0, 1],
-                               [0, 1]],
-    'tracked_shards': [],
+    "tracked_shard_schedule": [[0], [0], [1], [1]],
```

Comment:
This schedule seems a bit simpler than the old one, does it still cover the interesting cases?

Reply:
Yes:
* both shards are non-empty
* the second node does perform catchup
* downloading headers takes a moment that takes it to the next epoch